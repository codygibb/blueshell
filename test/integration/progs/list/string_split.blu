s := "hello this is a test string"
l := s.split(" ")
print l.len()
print l

file := ".tmp.txt"
$> echo "hello" > ${file}
$> echo "hello" >> ${file}
$> echo "hello" >> ${file}

data := $> cat ${file}

split_data := data.split('\n')
print split_data
print split_data.len()
print split_data[2]

$> rm ${file}
