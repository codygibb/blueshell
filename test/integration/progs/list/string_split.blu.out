6
["hello", "this", "is", "a", "test", "string"]
["hello", "hello", "hello"]
3
hello
